doc/users/plotting/colormaps/colormaps.py

```diff
@@ -7,14 +7,15 @@
                              'GnBu', 'Greens', 'Greys', 'Oranges', 'OrRd',
                              'PuBu', 'PuBuGn', 'PuRd', 'Purples', 'RdPu',
                              'Reds', 'YlGn', 'YlGnBu', 'YlOrBr', 'YlOrRd']),
-         ('Sequential (2)', ['afmhot', 'autumn', 'bone', 'cool',
-                             'copper', 'gist_heat', 'gray', 'hot',
-                             'pink', 'spring', 'summer', 'winter']),
+         ('Sequential (2)', ['afmhot', 'autumn', 'bone', 'cool', 'copper',
+                             'gist_heat', 'gray', 'hot', 'pink', 'spring',
+                             'Wistia', 'summer', 'winter']),
```

Comment:
See 3c29961: I tried to group the colormap in a more logical or hue-base fashion, rather than per alphabetical order.